### DISCUSSION TITLE
Is there multi-lingual support for NeMo?

### DISCUSSION THREAD

**Discussion by ramrahu** [NVIDIA-NeMo/NeMo#2074] [Q&A] [answered]
Hello,

Does NeMo ASR support Hindi and other languages? Will there be any tutorial/doc released any time soon on how to use NeMo ASR for multi lingual training?

**Comment by okuchaiev** [comment, marked as answer]
yes, we support other languages https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/asr/results.html 
But at the moment, we don't have Hindi ASR model.

**Comment by kruthikakr** [comment]
Is it possible to train single ASR model with multi languages using Nemo ?
